### PR TITLE
Add pure swift command line demo project to show the featurevisor in action

### PR DESCRIPTION
**Pull request by polok**
Add the possibility to easily see the SDK in action, test your config file or test the integration.

### CODE REVIEW

File: FeaturevisorDemo/FeaturevisorDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,322 @@
+// !$*UTF8*$!
```
Comment (fahad19) [inline FeaturevisorDemo/FeaturevisorDemo.xcodeproj/project.pbxproj:1]: @polok: is this a must-have to keep in the repo? or can we `.gitignore` it?

I haven't run any serious Xcode project before, and don't know myself if this is a common practice in the Swift world. I will trust your judgement here.

File: FeaturevisorDemo/FeaturevisorDemo/main.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import FeaturevisorSDK
+import FeaturevisorTypes
+
+print("Hello, Featurevisor!")
+
+let instanceOptions = InstanceOptions(
+        datafileUrl: "https://your-datafile.json",
+        logger: createLogger())
+
+let featurevisor = createInstance(options: instanceOptions)
```
Comment (fahad19) [inline FeaturevisorDemo/FeaturevisorDemo/main.swift:18]: if `FeaturevisorDemo` treated as a package or a separate app by having it as a different folder?

is it something worth recreating as a test case instead?

File: FeaturevisorDemo/FeaturevisorDemo.xcodeproj/project.pbxproj (hunk 1 above)
Comment (polok) [reply FeaturevisorDemo/FeaturevisorDemo.xcodeproj/project.pbxproj:1]: Well, it's autogenerated by Xcode. It is responsible for maintaining references to all of the linked files and their groupings, linked frameworks, and most importantly, the project’s build settings. Without it, we won't be able to run it.

File: FeaturevisorDemo/FeaturevisorDemo/main.swift (hunk 2 above)
Comment (polok) [reply FeaturevisorDemo/FeaturevisorDemo/main.swift:18]: I added it as a separate project to easily run and debug the SDK. Such demo projects are usually in a root folder of library/framework. However, now I think that for now, I won't push it. I will keep it locally and later we will decide. I will close this PR

**Comment by polok** [comment]
As discussed above, closing it for now

**Pull request closed without merging**